plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/BaseDeltaLakeConnectorSmokeTest.java

```diff
@@ -1538,4 +1546,79 @@ private Session broadcastJoinDistribution(boolean dynamicFilteringEnabled)
                 .setSystemProperty(ENABLE_DYNAMIC_FILTERING, Boolean.toString(dynamicFilteringEnabled))
                 .build();
     }
+
+    @Test
+    public void testDynamicFilterIsApplied()
```

Comment:
I think we already covered basic DF testing in TestDeltaLakeDynamicFiltering
Is there anything here that wasn't covered by that ?

Reply:
The only difference to the existing test is the underlying storage (ADLS, S3) on which the data is found in the concrete implementations of `BaseDeltaLakeConnectorSmokeTest`.

@findepi WDYT?

Reply:
We don't need to care about the filesystem on which the data is stored for DF tests.